hudi-spark-datasource/hudi-spark-common/src/main/scala/org/apache/spark/sql/hudi/command/AlterHoodieTableRenameCommand.scala

```diff
@@ -52,8 +52,7 @@ case class AlterHoodieTableRenameCommand(
       // update table properties path in every op
       if (hoodieCatalogTable.table.properties.contains("path")) {
         val catalogTable = sparkSession.sessionState.catalog.getTableMetadata(newName)
-        val path = catalogTable.storage.locationUri.get.getPath
-        AlterTableSetPropertiesCommand(newName, Map("path" -> path), isView).run(sparkSession)
+        AlterTableSetPropertiesCommand(newName, Map("path" -> catalogTable.location.toString), isView).run(sparkSession)
```

Comment:
It's great if we can have a test case.

Reply:
Added some checks in existing `TestAlterTable#Test Alter Table`